src/Symfony/Bridge/Propel1/Form/ChoiceList/ModelChoiceList.php

```diff
@@ -14,8 +14,6 @@
 use Symfony\Bridge\Propel1\Form\Type\ModelType;
 use Symfony\Component\Form\Exception\StringCastException;
 use Symfony\Component\Form\Extension\Core\ChoiceList\ObjectChoiceList;
-use Symfony\Component\Form\Extension\Core\DataTransformer\ChoiceToValueTransformer;
-use Symfony\Component\Form\Extension\Core\DataTransformer\ChoicesToValuesTransformer;
```

Comment:
this is bad, code uses those imports in documentation

fabbot.io is already taking care of unused imports